ix-dev/community/lms/templates/docker-compose.yaml

```diff
@@ -0,0 +1,26 @@
+{% set tpl = ix_lib.base.render.Render(values) %}
+
+{% set c1 = tpl.add_container(values.consts.lms_container_name, "image") %}
+
+{% do c1.add_caps(["CHOWN", "SETGID", "SETUID"]) %}
+
+{% do c1.ports.add_port(9000, values.network.web_port) %}
+{% do c1.ports.add_port(9090, values.network.cli_port) %}
+{% do c1.ports.add_port(3483, 3483) %}
+{% do c1.ports.add_port(3483, 3483, {"protocol": "udp"}) %}
+
+{% do c1.add_storage("/config", values.storage.config) %}
+{% do c1.add_storage("/music", values.storage.music) %}
+{% do c1.add_storage("/playlist", values.storage.playlist) %}
+{% for store in values.storage.additional_storage %}
+  {% do c1.add_storage(store.mount_path, store) %}
+{% endfor %}
+
+{% do c1.environment.add_env("PUID", values.consts.run_as_user) %}
+{% do c1.environment.add_env("PGID", values.consts.run_as_group) %}
```

Comment:
once you add the run_as section in both the test file and the questions, you wont need those. it will pick it up automatically

Reply:
also add the line to add additional env variables.